src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -810,10 +814,19 @@ export default function Swap({
                     <Price trade={trade} theme={theme} showInverted={showInverted} setShowInverted={setShowInverted} />
                   )}
 
-                  {!isExpertMode && !allowedSlippage.equalTo(INITIAL_ALLOWED_SLIPPAGE_PERCENT) && (
-                    <RowSlippage allowedSlippage={allowedSlippage} fontSize={12} fontWeight={400} rowHeight={24} />
+                  {((!isExpertMode && isUserNativeEthFlow) ||
```

Comment:
I am wondering whether we should show the slippage no matter whether we are in expert mode or not.

Just to warn a user that potentially there might be higher slippage than expected. Otherwise, users might not expect such a high slippage and if they are really suffering the higher slippage they might be very disappointed.

I know, usually one wants to abstract as much information away from the user as possible, but here, we might need the warning. What do you think?
I would also be interested in @MindyCoW  opinion: Should we show the higher slippage also in non-expert mode?

Reply:
i would be inclined to agree here actually. was thinking the same as i reviewed this code. will implement